addons/mail/static/src/models/notification_request_view.js

```diff
@@ -1,12 +1,29 @@
 /** @odoo-module **/
 
 import { registerModel } from '@mail/model/model_core';
-import { one } from '@mail/model/model_field';
+import { attr, one } from '@mail/model/model_field';
+
+import { sprintf } from '@web/core/utils/strings';
```

Comment:
Is it usual to add a new line to separate the imports of different modules? I don't remember.

Reply:
Yes